### PR TITLE
added the transition

### PR DESCRIPTION
**Pull request by Farnazmdi**
image
image

@aarmey The dependency installs have the same error as I used to get before you installed `pygraphviz` on the server.

### CODE REVIEW

**Review by aarmey — CHANGES_REQUESTED** [review]
Think I just fixed that. The numbers on the graph start at 1 (which I think is better), while the state numbers in the plots start at 0. Can you make these consistent?

**Comment by Farnazmdi** [comment]
Absolutely, will do right away.

**Comment by Farnazmdi** [comment]
@aarmey The installations for tests fails for the same reason. Could you please let me know how you fixed that for the build? 
image

**Comment by aarmey** [comment]
I think I had just rerun build but not test.

File: lineage/figures/cartoons/gemcitabine.svg [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
+ "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```
Comment (aarmey) [inline lineage/figures/cartoons/gemcitabine.svg:3]: If these are programmatically generated, they shouldn't be included in the repo.

**Review by aarmey — CHANGES_REQUESTED** [review]
Ok, I think I've finally fixed this.

File: lineage/figures/cartoons/gemcitabine.svg (hunk 1 above)
Comment (Farnazmdi) [reply lineage/figures/cartoons/gemcitabine.svg:3]: Thanks so much.
Yes, they are generated. Is there a way to remove them after they are called in `genFigures`?

Comment (aarmey) [reply lineage/figures/cartoons/gemcitabine.svg:3]: Look for how you can delete a file using Python.

**Review by aarmey — APPROVED** [review]

**Pull request merged**